Update openmrs.md: remove 'integration' (#1907)
medic/cht-docs: content/en/building/interoperability/openmrs.md

  - /building/guides/interoperability/openmrs
---

[OpenMRS](https://openmrs.org) is an open-source electronic medical record system used in dozens of countries. Data exchange between CHT apps and OpenMRS can open up opportunities to improve health outcomes for patients by promoting better coordination of care. For example, referrals by CHWs in the community can be sent electronically to health facilities using OpenMRS so that nurses and clinicians can prepare for their visit and have full access to the assessment done by a CHW.

> CHT apps can connect with OpenMRS using the the CHT's [Interoperability Tools]({{< ref "building/interoperability/openhim" >}}) and following the guidance in the [Building interoperability with FHIR APIs documentation]({{< ref "building/interoperability/cht-config" >}}). 

Reviewer: ```suggestion
CHT apps can exchange data with OpenMRS using the CHT's [Interoperability Tools]({{< ref "building/interoperability/openhim" >}}) and following the guidance in the [Building interoperability with FHIR APIs documentation]({{< ref "building/interoperability/cht-config" >}}). 
```

| `OPENMRS_HOST`            | hostname of OpenMRS instance                                                                    |
| `OPENMRS_PORT`            | port where OpenMRS FHIR API is listening                                                        |
| `OPENMRS_PASSWORD`        | OpenMRS password to use for basic authentication                                                |

> | `OPENMRS_USERNAME`        | OpenMRS password to use for basic authentication. Should be a user  created specifically for this connection. |

Reviewer: ```suggestion
| `OPENMRS_USERNAME`        | OpenMRS password to use for basic authentication. Should be a user created specifically for this connection |
```

{{< figure src="mediators.png" link="mediators.png" >}}

* The CHT Mediator Channel contains routes to the CHT Mediator.

> * The FHIR Channel contains routes to the FHIR Server. While this implementation does not utilize it, the channel can still expose any CHT documents sent to it as a FHIR API.

Reviewer: ```suggestion
* The FHIR Channel contains routes to the FHIR Server. While this implementation does not utilize it, the channel can expose any CHT documents sent to it as a FHIR API.
```
